partcad-cli/src/partcad_cli/click/commands/import/assembly.py

```
@@ -0,0 +1,31 @@
+    project = ctx.get_project("")
+    if not project:
+        pc_logging.error("Project retrieval failed.")
+        raise click.ClickException("Failed to retrieve the project.")
+
+    config = {"desc": desc} if desc else {}
+
+    try:
+        assy_name = import_assy_action(project, file_type, assembly_file, config)
+        click.echo(f"Assembly '{assy_name}' imported successfully.")
+    except Exception as e:
+        pc_logging.exception(f"Error importing assembly")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)